Content.Server/GameObjects/Components/Doors/AirlockComponent.cs

```diff
@@ -87,6 +101,19 @@ private void UpdatePowerCutStatus()
                                       _wires.IsWireCut(Wires.BackupPower);
         }
 
+        private void UpdateBoltedStatus()
```

Comment:
This whole method doesn't seem quite right.

When airlocks are bolted they don't show the deny light animation, they show a permanent bolt light.
(Which, speaking of, is what the "BLTL" wire is for, whether the light is visible or not)

Doing the above would also allow you to avoid the rather weird "reset appearance" in the else block.

Reply:
I was wondering what the BLTL was used for. Guess that makes sense